README.md

```diff
@@ -112,6 +112,30 @@ _must_ detect whether they're present or not before using them. See [this issue]
 and [this wiki page](https://github.com/silverstripe-australia/silverstripe-queuedjobs/wiki/Defining-queued-jobs) for 
 more information
 
+## Memory limit configuration
+
+By default this task will run until it comes without 12MB of the memory limit configured by PHP, giving the application
```

Comment:
This sentence is confusing. Do you mean it will stop before it has 12MB left, or will work until it has used 12MB. And do you mean 128MB?